core/src/main/java/org/apache/iceberg/deletes/EqualityDeleteWriter.java

```diff
@@ -43,6 +43,7 @@
   private final int[] equalityFieldIds;
   private final SortOrder sortOrder;
   private DeleteFile deleteFile = null;
+  private long rowOffset = 0;
```

Comment:
How about name `recordCount` or `rowCount`?
`rowOffset` is specific for tracking record write to file, `recordCount` is more general.

Reply:
I see the `BaseFile` is using the `recordCount`,  I think it's better to keep this consistent with the `BaseFile`'s `recordCount` :-)